bundles/org.openhab.binding.tr064/src/main/java/org/openhab/binding/tr064/internal/soap/SOAPConnector.java

```diff
@@ -186,7 +216,9 @@ public void sendChannelCommandToDevice(Tr064ChannelConfig channelConfig, Command
                                     channelConfig.getChannelTypeDescription().getGetAction().getParameter().getName(),
                                     parameter);
                         }
-                        doSOAPRequest(service, channelTypeDescription.getSetAction().getName(), arguments);
+                        SOAPRequest soapRequest = new SOAPRequest(service,
+                                channelTypeDescription.getSetAction().getName(), arguments);
```

Comment:
@cpmeister This is the only place where set actions take place and it bypasses the cache by calling `doSOAPRequestUncached`